docker/arm.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM arm64v8/alpine:3.16.2
+
+RUN apk add --no-cache gettext curl nginx php8 php8-fpm php8-opcache php8-pdo php8-pdo_sqlite php8-openssl && \
+    mkdir /var/www/html
+
+COPY . /var/www/html
+
+RUN chmod -R 775 /var/www/html && \
+    chown -R nginx:nginx /var/www/html
+
+COPY docker/php-fpm.conf /etc/php8/php-fpm.conf
```

Comment:
/etc/php81 here after the other two comments.

Reply:
See latest commits